src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
+  "minecolonies:shepherd.skills.desc": "Influences the amount of wool they obtain and the damage they deal.",
+  "minecolonies:cowboy.skills.desc": "Influences their working speed and the damage they deal.",
+  "minecolonies:swineherder.skills.desc": "Influences their working speed and the damage they deal.",
+  "minecolonies:chickenherder.skills.desc": "Influences the chance they succesfully pickup chicken byproducts and the damage they deal.",
+  "minecolonies:ranger.skills.desc": "Influences their speed, accuracy and damage output.",
+  "minecolonies:knight.skills.desc": "Influences their health and resilience.",
+  "minecolonies:composter.skills.desc": "Influences the working speed.",
+  "minecolonies:student.skills.desc": "No skill based bonuses.",
+  "minecolonies:archertraining.skills.desc": "No skill based bonuses.",
+  "minecolonies:combattraining.skills.desc": "No skill based bonuses.",
+  "minecolonies:sawmill.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:blacksmith.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:stonemason.skills.desc":"Influences the recipe improvement chance and working speed.",
+  "minecolonies:stonesmeltery.skills.desc": "Influences working speed and smelting speed.",
+  "minecolonies:crusher.skills.desc": "Influences working speed.",
+  "minecolonies:sifter.skills.desc": "Influences the working speed and chance to obtain items.",
+  "minecolonies:florist.skills.desc": "Influences the working speed.",
+  "minecolonies:enchanter.skills.desc": "Influences working speed and chance to obtain good enchantments.",
+  "minecolonies:researcher.skills.desc": "Influences the offline time researchers can leverage.",
+  "minecolonies:healer.skills.desc": "Influences the working speed and miraculous healing chance.",
+  "minecolonies:pupil.skills.desc": "No skill based influence.",
+  "minecolonies:teacher.skills.desc": "Influences the speed at which pupils learn.",
+  "minecolonies:dyer.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:glassblower.skills.desc": "Influences working speed and smelting speed.",
+  "minecolonies:mechanic.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:fletcher.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:planter.skills.desc": "Influences working speed.",
+  "minecolonies:rabbitherder.skills.desc": "Influences working speed and chance to hit a rabbit.",
+  "minecolonies:concretemixer.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:beekeeper.skills.desc": "Influences working speed and bee retaliation chance.",
+  "minecolonies:cookassistant.skills.desc": "Influences the recipe improvement chance and working speed.",
+  "minecolonies:smelter.skills.desc": "Influences the smelting and working speed.",
```

Comment:
smelting speed and chance of doubling/tripling ore

Reply:
Is this still true in 1.18? (Now that the smelter uses fortune to break down ore and doesn't double anymore)